### PR TITLE
WebKit export of https://bugs.webkit.org/show_bug.cgi?id=252373

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

**Review by nt1m — APPROVED** [review]

**Comment by sideshowbarker** [comment]
@nt1m well I only now notice that I’d left out the old/newState asserts from the details6 _“Calling open twice on 'details' fires only one toggle event”_

To fix that, what should I do first?

- First do a follow-up WebKit PR for https://github.com/WebKit/WebKit/pull/16795 that adds the assertions to `LayoutTests/imported/w3c/web-platform-tests` in WebKit — and then (re)export them here, in a follow-up WPT PR? or
- First do a follow-up WPT PR that adds the assertions here, then (re)import then to WebKit, in a follow-up WebKit PR?

I guess the advantage of doing the WebKit PR first would be that the updated tests get run by the EWS bots, so we know that they’re passing in WebKit.

(I do already know that _“Calling open twice on 'details' fires only one toggle event”_ is in fact passing with those asserts included — I think I’d already included those asserts in an interim commit in the WebKit PR branch, but then while I was troubleshooting locally to try to isolate what might be causing the flakiness for that particular test, I’d inadvertently ended up amending and force-pushing a local version in a state that omitted those assertions.)

**Comment by sideshowbarker** [comment]
@nt1m Also by the way, when I ran `export-w3c-test-changes` it didn’t work completely as expected.

I ran it like this:
```
export-w3c-test-changes -c -g HEAD -b 252373 --no-linter -u git@github.com:web-platform-tests/wpt.git -r wpt
```
…and that created the PR here as expected — except that it didn’t cause wpt-pr-bot to add the expected _“This patch has been exported from WebKit; it will be approved automatically once the downstream patch is r+”_ comment.

I guess maybe that part gets done correctly if `export-w3c-test-changes` detects that I’m a WebKit committer? Or maybe at least a WebKit contributor? (I’m not actually even in the WebKit contributors.json file yet.)

Or maybe it failed because I used `--no-linter` and that had the effect of suppressing some necessary step?

I had added `--no-linter` because when I tried _without_ adding that, I got this error:
> ERROR:lint:/opt/webkit/WebKitBuild/w3c-tests/web-platform-tests/html/semantics/interactive-elements/the-details-element/toggleEvent.html: /opt/webkit/WebKitBuild/w3c-tests/web-platform-tests/html/semantics/interactive-elements/the-details-element/toggleEvent.html matches an ignore filter in .gitignore - please add a .gitignore exception (IGNORED PATH)

…which as far as I could see was a spurious error — because that test _doesn’t_ actually match anything in .gitignore, and I think we definitely don’t want to add an `IGNORED PATH` exception for it…

**Pull request merged**
